### ISSUE TITLE
Two lines elements

### ISSUE THREAD

**Issue by produit** [nanograv/PINT#344]
Hi,

I would like to be able to use two lines elements for spacecraft position.
There is a nice python library (sgp4-1.4 https://pypi.python.org/pypi/sgp4/)
that is able to compute x,y,z and vx,vy,vz from time and tle elements.
Probably this library is redundant with code that we can find in astropy so could be
locally rewritten, but this library is well tested.
tle elements exists for example for FERMI and for POLAR. I don't know how accurate they are
but this would be a good occasion to compare.
I can code downloading tle files from www.space-track.org, finding the valid tle for 
a specified spacecraft and a specified time period and creating spacecraft position structures.
my questions:
is it ok to add a new dependence (sgp4) and how?
Is anybody interested?
I think it is a good exercise for me but I will need some directions and help...

**Comment by luojing1211** [comment]
If you want to use sgp4 with PINT, check out the observatory module. The file : PINT/pint/observatory/fermi_obs.py  is a good example. 
But adding this dependence into PINT officially needs more discussion.

**Comment by paulray** [comment]
Yes, I think making an observatory special location "TLE" would be good. It would very much parallel the other observatories, as @luojing1211 noted. It would just compute state vectors from a TLE instead of reading from a tabulated ephemeris.

**Comment by aarchiba** [comment]
Presumably it should permit TLE data to change from epoch to epoch? With some kind of interpolation?

Is the interface of the PINT observatories stable and flexible enough that a third-party module could be written (that is, outside PINT) and used?

**Comment by aarchiba** [comment]
TLE support might resolve interpolation problems with orbit files - see #434 - particularly if TLEs can be fitted to XYZ orbital tracks.

**Comment by aarchiba** [comment]
I notice that sgp4 reports that satellites move off their TLE positions by 1-3 km/day, so accuracy is going to be limited.
